src/lib/y2users/password.rb

```diff
@@ -18,20 +18,20 @@
 # find current contact information at www.suse.com.
 
 require "yast2/execute"
+require "yast2/secret_attributes"
 
 module Y2Users
   # Password configuration for user including its hashed value.
   class Password
+    include Yast2::SecretAttributes
+
     # @return [String] login name for given password
     attr_reader :name
 
-    # @return [String, nil] Encrypted password. It can have several specific values:
-    #   - "!" or "*" is disabled login by password
-    #   - "" password-less login allowed
-    #   - "!..." disabled password. After exclamation mark is old value that no longer can be used
-    #     for login
-    #   - nil means password is not yet set
-    attr_reader :value
+    # @!attribute value
+    # @return [Value, nil] password value. Can be any subclass of Value. nil when password is
+    #   not set at all.
+    secret_attr :value
```

Comment:
Does this need to be a `secret_attr`? I think that making `PasswordValue::content` secret is enough. I suspect this adds nothing but noise.

Reply:
To be honest I am not sure. Question is if it is secret if value is nil?